Makefile

```diff
@@ -103,6 +103,12 @@ Q := $(if $V,,@)
 	$Q ln -s ../bin .GOPATH/bin
 	$Q touch $@
 
+.GOPATH/.setup:
```

Comment:
It could be, though as setup depends on .GOPATH/.ok then it would in all likelihood complain about not running setup when you're actually running setup. I'll have a play with that now though and get back to you.